webpack.config.dev.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const HTMLLoader = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.tsx',
+  mode: 'development',
+  output: {
+    filename: "[name].js",
```

Comment:
Add hashing for js files to invalidate browser cash when you add changes to you app